ank/src/cli_commands/dry_run_plan.rs

```diff
@@ -0,0 +1,123 @@
+use std::collections::{BTreeMap, BTreeSet};
+use tabled::Tabled;
+use super::cli_table::CliTable;
+use common::objects::WorkloadInstanceName;
+
+#[derive(Clone, Debug)]
+pub enum DryRunAction {
+    NoChange,
+    Create,
+    Recreate,
+    Delete,
+}
+
+impl std::fmt::Display for DryRunAction {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let s = match self {
+            DryRunAction::NoChange => "no change",
+            DryRunAction::Create   => "create",
+            DryRunAction::Recreate => "recreate",
+            DryRunAction::Delete   => "delete",
+        };
+        write!(f, "{s}")
+    }
+}
+
+#[derive(Clone, Debug, Tabled)]
+#[tabled(rename_all = "UPPERCASE")]
+pub struct DryRunPlanRow {
+    #[tabled(rename = "WORKLOAD")]
+    pub workload: String,
+    #[tabled(rename = "ACTION")]
+    pub action: String,
+    #[tabled(rename = "DETAILS")]
+    pub details: String,
+}
```

Comment:
Removing the details column (reason explained in the first comment of this file):
```suggestion
#[derive(Clone, Debug, Tabled)]
#[tabled(rename_all = "UPPERCASE")]
pub struct DryRunPlanRow {
    #[tabled(rename = "WORKLOAD")]
    pub workload: String,
    #[tabled(rename = "ACTION")]
    pub action: String,
}
```